parfive/downloader.py

```diff
@@ -372,28 +374,120 @@ def callback(token, future, main_pb):
                                           total=get_http_size(resp))
                     else:
                         file_pb = None
-                    with open(str(filepath), 'wb') as fd:
-                        while True:
-                            chunk = await resp.content.read(chunksize)
-                            if not chunk:
-                                # Close the file progressbar
-                                if file_pb is not None:
-                                    file_pb.close()
-
-                                return str(filepath)
 
-                            # Write this chunk to the output file.
-                            fd.write(chunk)
+                    # setup asyncio queue and writer
```

Comment:
I don't think this comment has a huge amount of value, either use it to describe what this queue is used for or remove it please.